kythe/release/README.md

```diff
@@ -22,6 +22,7 @@ extractors, and tools directly supported by the Kythe team.
    - go_extractor             :: Go extractor
    - javac-wrapper.sh         :: javac wrapper script for extractor
    - javac_extractor.jar      :: Java extractor
+   - extract_kzip             :: Generic kzip extractor
```

Comment:
Can you explain a little more what this binary does?

Reply:
That at least matches to other tool descriptions...